sandbox/cgroup/scanning.go

```diff
@@ -0,0 +1,155 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package cgroup
+
+import (
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/snapcore/snapd/snap/naming"
+)
+
+var (
+	// string that looks like an hook security tag
+	taggish1 = regexp.MustCompile(`snap\.[^.]+\.hook\.[^.]+`)
```

Comment:
that's not a great name,   hookSecurityTagRoughPattern,  ...RoughDetector ?

also because when I scanned this first I thought 1 and 2 corresponded to v1 v2

Reply:
Indeed, I didn't think about the cgroup name association.